### ISSUE TITLE
Accessibility-related "Skip to Content" button doesn't work as expected

### ISSUE THREAD

**Issue by pb-amy** [pressbooks/pressbooks#2128]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing this issue with one of our built-in root or book themes?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Description

This issue was mentioned in this client interaction: https://pressbooks.zendesk.com/agent/tickets/4704

Wordpress has a native function that allows you to 'tab' through linked content, starting with a button at the top called "Skip to Content".

After pressing enter on "Skip to Content", you would be taken to the content of the page. From here, upon pressing 'tab', each time, you'd ideally be be sequentially taken through all of the links throughout the page.

What happens in actuality is that when you press enter on "Skip to Content", it takes you to the content. However, upon pressing tab, it takes you back to the "Skip to Content" button at the top of the page instead of taking you to the next link on the page, forcing the user to re-navigate down the page again. 

You can see this on this [video](https://drive.google.com/file/d/1gjFb0J9GN0H7KEH6NrxvsV_jCwdilxzM/view) - I press 'tab' when the video starts, which enables the "Skip to Content" button to show. Then I press enter, which takes me to content (good), but once I'm on my content, I press 'tab' again, and it takes me back up to "Skip to Content" instead of the next link in my chapter. FYI, this forum discussion describes exactly what happens on Pressbooks: https://wordpress.org/support/topic/skip-to-content-5/

### Steps to Reproduce

1. Press 'Tab' on any Pressbooks chapter
2. Press 'Enter', once "Skip to Content" appears
3. You will be taken down to the content. Press 'Tab' again.

**Expected behavior** After step 3, you should be taken to the next link in the content

**Actual behavior:** After step 3, you are taken back to the "Skip to Content" button.
